src/path/ancestors.rs

```diff
@@ -21,7 +21,7 @@ use crate::path::Path;
 ///
 /// [`ancestors`]: struct.Path.html#method.ancestors
 /// [`Path`]: struct.Path.html
-#[derive(Copy, Clone, Debug)]
```

Comment:
This is a breaking change. What is the reasoning?

Reply:
I think this might have been a mistake, I'll revert.